src/custom-elements/src/components/expansion-panel/expansion-panel.tsx

```diff
@@ -34,6 +38,11 @@ export class ExpansionPanel {
    */
   @Prop({ reflect: true }) stateIcon = false;
 
+  /**
+   * to render a state tooltip
```

Comment:
```suggestion
   * To render a state tooltip
```

Reply:
@josecarlosgonzalezv  why should it be capitalized here if for other properties it's lower-cased